### ISSUE TITLE
Integration with eAccounting API

### ISSUE THREAD

**Issue by thor** [SPF-UiO/spbm#21]
This is a bigger one. 

For invoicing, we want to integrate with the eAccounting API for us to achieve two things, for the most part:
1. Ensure correct creation and detailed itinerary of invoices sent to participating personnel organisations
2. Ensure correct payment and subsequent data basis for payroll executions

The latter is particularly useful, as we want to reduce the dependency of organisation representatives on the financial administrator.

**Comment by maxiix3** [comment]
har vi etterspurt apikey fra visma sånn at man faktisk kan gjøre noe her?

**Comment by thor** [comment]
Ja, absolutt – problemet var tid og det å få et fungerende oppsett med feks codegen og Swagger/CoreAPI.

Si ifra hva du trenger om du vil se på det, ev. kan jeg legge det ut på wikien, eller forespørre nytt fra de.

Av funksjonalitet er det kun fakturering og synkroniseringen der som er relevant.
